config/config.go

```diff
@@ -0,0 +1,83 @@
+/*
+// Copyright (c) 2016 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+*/
+
+package config
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+)
+
+type CIAOConfig struct {
+	UsedFile  string
+	Launcher  LauncherConfig
```

Comment:
You could embed LauncherConfig.  This would result in slightly shorted code in main.go, e.g.,

```
flag.StringVar(&serverURL, "server",  defaults.Server, "URL of SSNTP server")  
```
instead of
```
flag.StringVar(&serverURL, "server", defaults.Launcher.Server, "URL of SSNTP server")  
```